genyrator/templates/sqlalchemy/model/sqlalchemy_model.j2

```diff
@@ -2,6 +2,10 @@ from sqlalchemy_utils import UUIDType
 from sqlalchemy import UniqueConstraint
 from sqlalchemy.types import JSON as JSONType
 
+# Available for custom sqlalchemy_options
```

Comment:
could this be a comment in the template instead of the generated module?

Reply:
The reason I put it in the generated module is because we have come across the situation where it's not clear why something is in the generated module. As a client of the library it would be useful to have some indication of why things exist without having to read the source code of the library I'm depending on.

Reply:
I guess that's the trade-off, either we have verbose generated code or we make the generated code harder to follow. let's just be sure we are consistent